src/providers/WorkspaceSymbolProvider.ts

```diff
@@ -2,7 +2,7 @@ import * as vscode from "vscode";
 import { AtelierAPI } from "../api";
 import { ClassDefinition } from "../utils/classDefinition";
 import { DocumentContentProvider } from "./DocumentContentProvider";
-import { StudioOpenDialog } from "../queries";
```

Comment:
Why did you decide to remove this and also the type declaration of the result from the StudioOpenDialog query later?

Reply:
@gjsjohnmurray The prior version of the query used "CALL". I switched it to "SELECT TOP 10 Name" because we only use the "Name" field. Should I revert to using CALL?

Reply:
Sorry, I hadn't spotted that difference. No, I don't think there's any need to revert to using CALL. Reducing to a single column the amount of data returned from the query makes good sense, as does the use of SELECT TOP 10 (though that probably results in a change of behaviour in some situations, i.e. fewer results). @daimor may have an opinion about whether to still declare the result as StudioOpenDialog (not quite accurate), or to export another interface from query.d.ts that only contains Name.

Reply:
@gjsjohnmurray @daimor Are there any further action items I should handle or are my changes sufficient?